source/vibe/web/rest.d

```diff
@@ -744,6 +747,20 @@ private HTTPServerRequestDelegate jsonMethodHandler(T, alias Func)(T inst, RestI
 				auto ret = handler(&__traits(getMember, inst, Method), params);
 				res.writeJsonBody(ret);
 			}
+		}
+
+		try {
+			static if (error_attributes.length == 1) {
+				auto Attr = error_attributes[0];
```

Comment:
Actually, it is, see next line. It's a workaround for a DMD bug: https://issues.dlang.org/show_bug.cgi?id=14889